HelpMyStreet.Utils/HelpMyStreet.Utils/Extensions/SupportActivityExtensions.cs

```diff
@@ -97,6 +107,9 @@ public static string PerfectTense(this SupportActivities activity, int count)
                     SupportActivities.MedicalAppointmentTransport => $"{count} people transported to medical appointments",
                     SupportActivities.ColdWeatherArmy => $"{count} cold weather army tasks completed",
                     SupportActivities.Other => $"{count} other tasks completed",
+                    SupportActivities.MealtimeCompanion => $"{count} meal time companion tasks completed",
```

Comment:
I would suggest:
* {} meals shared
* {} meals delivered
* {} volunteers supported

(and singular forms above)